src/window.zig

```diff
@@ -60,6 +61,14 @@ pub const Window = struct {
         self.peer.setTitle(title);
     }
 
+    pub fn setIcon(self: *Window, icon: *Image_Impl) void {
```

Comment:
I think it's redundant to use Image_Impl only to use .data just after. Seems much better to directly use ImageData
```suggestion
    pub fn setIcon(self: *Window, icon: *ImageData) void {
```